tests/bson/bson-timestamp-set_state_error-004.phpt

```diff
@@ -1,26 +1,24 @@
 --TEST--
-MongoDB\BSON\Timestamp::__set_state() with broken numeric strings
---SKIPIF--
-<?php if (8 !== PHP_INT_SIZE) { die('skip Only for 64-bit platform'); } ?>
```

Comment:
I believe this was erroneously copy-pasted when the test was created in d25e565f647d824ced02146bd41d8ef17ff93cb9.